packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -100,6 +98,10 @@ abstract class TransitionRoute<T> extends OverlayRoute<T> {
   /// the opaque route will not be built to save resources.
   bool get opaque;
 
+  /// This ensures that if we got to the dismissed state while still current,
```

Comment:
nit: should this be a `//` comment instead if `///`? I don't think it should appear in the docs.